### DISCUSSION TITLE
Ability to add additional games on demand

### DISCUSSION THREAD

**Discussion by asheroto** [seDirector/App#60] [Ideas]
Add custom games

seDirector supports `srcds.exe` games, so we may have to add support for other exes and parameters.

**Comment by Nomy61** [comment]
would like the abilty to add more steam games like Insurgency Sandstorm

**Comment by asheroto** [comment]
I've added this to the [Roadmap](https://github.com/seDirector/App/projects/2#card-89048151) and created an [Issue](https://github.com/seDirector/App/issues/62).
